apollo-ios-codegen/Sources/GraphQLCompiler/CompilationResult.swift

```diff
@@ -164,7 +160,7 @@ public class CompilationResult: JavaScriptObject {
 
     lazy var directives: [Directive]? = self["directives"]
 
-    public lazy var isDeferred: IsDeferred = getIsDeferred()
+    public lazy var deferCondition: DeferCondition? = getDeferCondition()
```

Comment:
When I merge my changes for 1.7.0 into `main` this will need to change a little. For concurrency, we need to make all the JavaScript types immutable, so no `lazy` vars. I'll make the change to update this to calculate and store the value during initialization.

So this is fine for this PR.